tests/scale/kube-burner/templates/pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: app-{{.Iteration}}-{{.Replica}}
+spec:
+  containers:
+    - name: app-{{.Iteration}}-{{.Replica}}
+      image: centos
```

Comment:
wondering if we can use `k8s.gcr.io/pause:3.6` image because
- centos image is already dead 
- to have faster image pull because of lesser in size
- no need to have infinite while loop